wagtail/admin/static_src/wagtailadmin/scss/userbar.scss

```diff
@@ -47,11 +48,15 @@ $positions: (
   ),
 );
 
+
+
```

Comment:
In the future, make sure to avoid adding spaces in places like this. It creates problems for us when pull requests include more changes than are actually needed.

Reply:
> In the future, make sure to avoid adding spaces in places like this. It creates problems for us when pull requests include more changes than are actually needed.

Ok, well noted